### ISSUE TITLE
NZXT (kraken and AER specifically) support?

### ISSUE THREAD

**Issue by PhonicSword** [DarthAffe/RGB.NET#18]
The CAM software they currently use is very buggy, and it would be nice to have their RGBs work in a different program. Don't know how hard it will be but thanks in advance!

**Comment by DarthAffe** [comment]
NZXT was already requested by someone and as far as I know there's no official SDK for it.
It might be possible since there are some other projects around it, but without a documented way it'll go way down on my priority list.

**Comment by dennisblokland** [comment]
https://github.com/KsenijaS/krakenx/blob/master/PROTOCOL.md is the protocol discription for x52 and x62

**Comment by SElben** [comment]
Would love to see my Kraken X52 rgb working together with my Asus MB and my MSI gpu rgbs

**Comment by akmadian** [comment]
@SElben It's still not possible as of now :/ NZXT hasn't made an SDK...

**Comment by Pet0203** [comment]
Isn't CAM communicating through serial? Wouldn't need an SDK if the protocol is known.

**Comment by akmadian** [comment]
Yeah as far as I know, CAM communicates over serial so we wouldn't need a proper SDK. However, the only protocol known as far as I know is for the krakens. Idk how difficult it would be to build the provider, but I don't have my hopes up...

**Comment by akmadian** [comment]
@PhonicSword I know this is from months ago, but I'm working on an NZXT SDK, so if you're looking for one, check here: https://github.com/akmadian/NZXTSharp

**Comment by PhonicSword** [comment]
@akmadian Thanks for the heads up! I actually saw your message when you sent it out, but was swamped by school and forgot about it until just now haha. Appreciate it!

**Comment by icanhazpython** [comment]
This may also be of use. Works for pump and RGB control of my nzxt kraken. 
https://github.com/jonasmalacofilho/liquidctl

**Comment by piechade** [comment]
Nothing new about NZXT, a native RGB.net would be nice. Ooenrgb has support for it.

**Comment by DarthAffe** [comment]
As there's no official sdk it's quite unlikely I'll add native support - it's just to much work to maintain especially without owning such devices.
If it's included in openRGB you could load them by using diogotr7s OpenRGB Device Provider https://github.com/diogotr7/RGB.NET.Devices.OpenRGB

**Comment by piechade** [comment]
If someone has interesting. That's how a Nzxt can look like https://github.com/piechade/Artemis.Plugins.Public/tree/master/src/Device/NZXT

**Comment by DarthAffe** [comment]
I'm planning on collecting 3rd party providers in a list somewhere - keeping that open for now :)

**Comment by piechade** [comment]
> I'm planning on collecting 3rd party providers in a list somewhere - keeping that open for now :)

I really net to get my Artemis setup working again. I fixed the Headset Logitech and NZXT problem. I want to add support for the display too. But there are some problems. Aura sync is not working on my gaming PC, it has problem with my Xiaomi USB light. On my working Laptop the USB light and monitor Aura sync is perfectly detected. In my gaming PC with Mainboard, RAM and graphics card it's crashing. Second thing I found adapter cable on ebay to connect NZXT area fans directly to the Mainboard. I can only recommend to buy this cabels and simply use the Mainboard software.
